src/airport.js

```diff
@@ -14,13 +14,13 @@ const airport = {
 
     isAirportFull: function () 
     {
-        return this.landedPlanes.length > this.defaultAirportCapacity
+        return this.landedPlanes.length < this.defaultAirportCapacity
```

Comment:
Glad I sought help from Ed, this little mistake was frustrating me as I was expecting the code to pass the test but it kept failing. Seeking "external" help allowed for different perspective on my code and it definitely helped me.